src/processes.c

```diff
@@ -2029,6 +2029,8 @@ static int ps_read (void)
 	int stopped  = 0;
 	int paging   = 0;
 	int blocked  = 0;
+	int num_cmdline  = 0;
```

Comment:
I think these lines try to align on the `=`, so this should have exactly 1 space before the `=`, and the rest should be adjusted to line up with this one.

Reply:
Done.